src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -944,18 +964,65 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! CallFailed(collector, None, e.getMessage)
       case Success(outputs) =>
         logger.info(s"Collection complete for Scattered Call ${collector.tag}.")
-        self ! CallCompleted(collector, outputs, 0)
+        self ! CallCompleted(collector, outputs, 0, "")
     }
 
     Success(ExecutionStartResult(Set(StartEntry(collector, ExecutionStatus.Starting))))
   }
 
+  private def sendStartMessage(callKey: CallKey, callInputs: Map[String, WdlValue]) = {
+    def registerAbortFunction(abortFunction: AbortFunction): Unit = {}
+    val backendCall = backend.bindCall(workflow, callKey, callInputs, AbortRegistrationFunction(registerAbortFunction))
+
+    /* Tries to use the cached Execution to send a UseCachedCall message.  If anything fails, send an InitialStartCall message */
+    def loadCachedCallOrInitiateCall(execution: Execution) = {
+      globalDataAccess.getWorkflow(execution.workflowExecutionId) onComplete {
```

Comment:
Another request to pull stuff out of an `onComplete`.  Also it seems like the two `InitialStartCall` cases should be able to be squashed into one condition